bootstrap/integration/bootstrap_tester.go

```diff
@@ -303,10 +303,13 @@ func (b *BootstrapTester) ReadEnvFromOutput(key string) (string, bool) {
 
 // Run the bootstrap and then check the mocks
 func (b *BootstrapTester) RunAndCheck(t *testing.T, env ...string) {
-	if err := b.Run(t, env...); err != nil {
-		t.Logf("Bootstrap output:\n%s", b.Output)
+	err := b.Run(t, env...)
+	t.Logf("Bootstrap output:\n%s", b.Output)
```

Comment:
This doesn’t pollute the test logs because the `gosum` test runner only shows an individual test’s output if the test ultimately fails. Changing this means that _any_ test failure will include the bootstrap’s logs, not just an exec failure on the bootstrap.